content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -77,7 +81,7 @@ gcloud projects remove-iam-policy-binding ${MANAGED_PROJECT} \
 To delete the Google service account, and the management cluster:
```

Comment:
```suggestion
To delete the Google service account and the management cluster:
```

Reply:
/cc @joeliedtke 
I've been following his advice and he suggests always adding the `,` there. I'm not so sure why though.

Reply:
You should [add commas after introductory words and phrases](https://developers.google.com/style/commas#commas-after-introductory-words-and-phrases). In this case, the entire sentence is an introductory phrase. For example, here is how the sentence would look if we added a bit more to it:

To delete the Google service account and the management cluster, use the following instructions:

Reply:
Thanks! I'll learn more about the rules